readme.md

```diff
@@ -52,29 +52,6 @@ The simple form for requesting a deploy can be seen here (further options are av
 ![Continuous deployment configuration](contrib/img/deployment_continuous.png)
 Riff-Raff polls our build server frequently and can be configured to automatically start a deploy for newly completed builds
 
-How do I run Riff-Raff locally if I want to hack on it?
```

Comment:
What do you think about linking to `CONTRIBUTING.md` from here to make it easily discoverable?